app.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Since this file is not part of this PR, we can remove it.

Reply:
Ok, I will do just that. thanks